allenact_plugins/ithor_plugin/ithor_environment.py

```diff
@@ -685,25 +685,25 @@ def step(
 
             sr = self.controller.step({"action": "GetReachablePositions"})
             self._initially_reachable_points = self.controller.last_event.metadata[
-                "reachablePositions"
+                "actionReturn"
             ]
             self._initially_reachable_points_set = None
             self.last_action = action
             self.last_action_success = metadata["lastActionSuccess"]
-            self.controller.last_event.metadata["reachablePositions"] = []
+            self.controller.last_event.metadata["actionReturn"] = []
         elif "RotateUniverse" in action:
             sr = self.controller.step(action_dict)
             metadata = self.last_event.metadata
 
             if metadata["lastActionSuccess"]:
                 sr = self.controller.step({"action": "GetReachablePositions"})
                 self._initially_reachable_points = self.controller.last_event.metadata[
-                    "reachablePositions"
+                    "actionReturn"
                 ]
                 self._initially_reachable_points_set = None
                 self.last_action = action
                 self.last_action_success = metadata["lastActionSuccess"]
-                self.controller.last_event.metadata["reachablePositions"] = []
+                self.controller.last_event.metadata["actionReturn"] = []
```

Comment:
What is the point of removing the reachable positions from `actionReturn`?

Reply:
It's not strictly necessary but should `RotateUniverse` or `RandomizeHideSeekObjects` shouldn't return reachable positions this is just making sure that no such points are available to to the user (we call `GetReachablePositions` here to refresh a cache of such positions in the `ITHOREnvironment`).